hermes/cmd/hermes.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+const (
+	flagConfig = "config"
+)
+
+// NewRelayer creates a new relayer command that holds
+// some other sub commands related to hermes relayer.
+func NewRelayer() *cobra.Command {
+	c := &cobra.Command{
+		Use:           "hermes [command]",
```

Comment:
I'm not sure about this but it might be worth considering a namespace for relayers.
For example `ignite relayer hermes` so we could have `ignite relayer ts` and potentially `ignite relayer go`.

PR #4 uses `ignite explorer gex` which would allow to add more explorers under the `ignite explorer` prefix.